### PR TITLE
[C-5561] adds audiences support

### PR DESCRIPTION
**Pull request by suhasdeshpande**
## Description of the change

implements `/audiences` endpoints 

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/java/services/AudiencesInterface.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package services;
+
+import models.*;
+import retrofit2.Call;
+import retrofit2.http.*;
+
+public interface AudiencesInterface {
+        @GET("/dev/audiences/{audienceId}")
```
Comment (tk26) [inline src/main/java/services/AudiencesInterface.java:8]:
```suggestion
        @GET("/audiences/{audienceId}")
```

File: src/main/java/services/AudiencesInterface.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        Call<Audience> getAudience(
+                        @Path("audienceId") String audienceId,
+                        @Header("Authorization") String authorization,
+                        @Header("User-Agent") String userAgent);
+
+        @PUT("/dev/audiences/{audienceId}")
```
Comment (tk26) [inline src/main/java/services/AudiencesInterface.java:14]:
```suggestion
        @PUT("/audiences/{audienceId}")
```

File: src/main/java/services/AudiencesService.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
+package services;
+import lombok.NonNull;
```
Comment (tk26) [inline src/main/java/services/AudiencesService.java:2]:
```suggestion

```

File: src/main/java/services/AudiencesInterface.java (hunk 1 above)
Comment (suhasdeshpande) [reply src/main/java/services/AudiencesInterface.java:8]: opsss!!

**Review by tk26 — APPROVED** [review]

**Pull request merged**